TransferMethod/Tests/View/Widget/TextWidgetTests.swift

```diff
@@ -0,0 +1,199 @@
+import HyperwalletSDK
+@testable import TransferMethod
+import XCTest
+
+class TextWidgetTests: XCTestCase {
+    private var textWidget: TextWidget!
+    private var inputText: String?
+    private var pattern: String?
+    private var expectedFormattedText: String?
+    private var scrubRegex: String?
+    private var expectedScrubbedText: String?
+
+    override func setUp() {
+        let coder = NSKeyedUnarchiver(forReadingWith: NSMutableData() as Data)
+        textWidget = TextWidget(coder: coder)
+    }
+
+    func testInputs() {
```

Comment:
rename to testFormatDisplayString()

Reply:
This tests both formatDisplayString and getScrubbedText

Reply:
Maybe `testFormatAndScrub`

Reply:
testFormatAndScrubText